Don't overwrite config if present

rudolfochrist opened the pull request:
I had multiple registries configured in `ocicl-registry.cfg` which were discarded after `setup` ran again while building a new executable. 

I think that, once the configuration was created, `ocicl` shouldn't touch it again and overwrite any existing configuration with default values.

What do you think @atgreen?

atgreen commented:
Thanks.  I've added a --force option to override the existing setting, and I've updated the docs.  Also, setup will tell you if it's preserving the registry settings.

rudolfochrist commented:
> Thanks. I've added a --force option to override the existing setting

Awesome!

> and I've updated the docs.

Even more awesome!

Thank you 👌
